src/lib/snark_bits/bits.ml

```diff
@@ -148,7 +148,7 @@ end
 
 module Snarkable = struct
   module Small_bit_vector
-      (Impl : Snarky.Snark_intf.S) (V : sig
+      (Tick : Tick_intf.S) (V : sig
```

Comment:
Can we leave the name as `Impl` here? This functor works with any snarky proof system, so it's a bit misleading to label it as if only `Tick` should be used.